package.json

```diff
@@ -134,5 +134,10 @@
     "stylus": "^0.54.8",
     "testcafe": "^2.6.2",
     "testcafe-browser-provider-saucelabs": "^1.9.0"
+  },
+  "overrides": {
```

Comment:
There is an option in NPM where you can tell it to override the versions defined in the dependencies `package.json` files.